stabilizer/src/test/java/com/hazelcast/stabilizer/AgentSmokeTest.java

```diff
@@ -150,7 +155,7 @@ public void run() {
     private AgentsClient getClient() throws IOException {
         File agentFile = File.createTempFile("agents", "txt");
         agentFile.deleteOnExit();
-        Utils.writeText("192.168.1.105", agentFile);
+        Utils.writeText(AGENT_IP_ADDRESS, agentFile);
```

Comment:
I think the best option for this test-scenario would be a support for 127.0.0.1, which doesn't work for some reason.